pyproject.toml

```diff
@@ -88,7 +109,6 @@ logger-objects = ["logHandler.log"]
 "winsound.PlaySound".msg= "PlaySound uses winmm, which is deprecated in favor of the Windows core audio APIs."
 
 [tool.licensecheck]
-using = "requirements:requirements.txt"
```

Comment:
this is missing packages from l10nUtil.exe which is bundled with NVDA. I think these are the docs and l10n groups

Reply:
Ah yes, I added them as well. I'think we're ok now.